xwiki-platform-core/xwiki-platform-search/xwiki-platform-search-solr/xwiki-platform-search-solr-ui/src/main/resources/Main/SolrSearchConfig.xml

```diff
@@ -51,6 +51,13 @@
     'OBJECT': 'objcontent',
     'OBJECT_PROPERTY': 'propertyvalue'
   },
+  'phraseFields': 'title_en^200.0 title__^200.0
+                   filename^45.0 
+                   doccontent_en^40.0 doccontent__^40.0
+                   attcontent_en^35.0 attcontent__^35.0 
```

Comment:
Same, you should use just "doccontent" and "attcontent", and they will get expanded.

Reply:
Same here. I couldn't get it to expand, hence why I hard coded it. If I didn't explicitly specify as I've done then the terms wouldn't appear at all in the query string as displayed in debug mode.